composer.json

```diff
@@ -235,9 +236,24 @@
         "spryker/product-offer-availabilities-rest-api": "^1.1.0",
         "spryker/product-offer-prices-rest-api": "^2.0.1",
         "spryker/product-offer-sales-rest-api": "^1.0.1",
+        "spryker/product-offer-service-point": "^1.0.0",
         "spryker/product-offer-service-point-availabilities-rest-api": "^1.0.0",
+        "spryker/product-offer-service-point-availability": "^1.0.0",
+        "spryker/product-offer-service-point-availability-calculator-storage": "^1.0.0",
+        "spryker/product-offer-service-point-availability-calculator-storage-extension": "^1.0.0",
+        "spryker/product-offer-service-point-availability-storage": "^1.0.0",
+        "spryker/product-offer-service-point-availability-storage-extension": "^1.0.0",
+        "spryker/product-offer-service-point-data-import": "^0.4.0",
+        "spryker/product-offer-service-point-gui": "^1.0.0",
         "spryker/product-offer-service-point-merchant-portal-gui": "^1.0.0",
+        "spryker/product-offer-service-point-storage": "^1.0.0",
+        "spryker/product-offer-shipment-type": "^1.0.0",
+        "spryker/product-offer-shipment-type-availability": "^1.0.0",
+        "spryker/product-offer-shipment-type-availability-storage": "^1.0.0",
+        "spryker/product-offer-shipment-type-data-import": "^0.4.0",
+        "spryker/product-offer-shipment-type-gui": "^1.0.0",
         "spryker/product-offer-shipment-type-merchant-portal-gui": "^1.0.0",
+        "spryker/product-offer-shipment-type-storage": "^1.0.0",
```

Comment:
No new modules should be added. (Nor full list commented) They are part of some feature already

Reply:
<img width="1172" alt="Screenshot 2023-11-03 at 21 50 48" src="https://github.com/spryker-shop/b2c-demo-marketplace/assets/15832795/525fc656-f751-4d4e-bc73-4c96d655a9fd">